src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
Element classes should follow the BEM methodology without combining modifiers and elements. Instead of 'stopwatch stopwatch--speed-up', you should have a base class for the block and use a modifier class if you need to represent a different state or variation of the block.